modules/data.land/R/soil_utils.R

```diff
@@ -20,16 +20,38 @@
 #' @return list of soil hydraulic and thermal parameters
 #' @export
 #' @importFrom rlang %||%
-    # null-filling operator: A %||% B == "B if A is null, A in all other cases"
+# null-filling operator: A %||% B == "B if A is null, A in all other cases"
```

Comment:
Nit: I preferred the space here, to visually distinguish this comment from the Roxygen block around it.
Though I do admit that interleaving `#'` and `#` like this is always ripe for confusion and maybe is a sign this needs a bigger rewrite :)
```suggestion
    # null-filling operator: A %||% B == "B if A is null, A in all other cases"
```

Reply:
The `%||%` operator is in base r-devel and will be in the next big release of R! (still good to import from rlang to not require a min R version though)

Reply:
@Aariq This is good news! Would you agree that means it's well-known enough we don't need to explain the import and could delete this comment entirely?

Reply:
Haha, no, I think it's actually really confusing and not obvious what it does at all 😅.  I find that comment very helpful! However, it might make more sense to move it to wherever in the function(s) `%||%` is first used rather than mixing roxygen comments (which _do_ something) and comments about those comments.

Reply:
Good call -- will do that below.